src/current/molt/migrate-failback.md

```diff
@@ -5,54 +5,131 @@ toc: true
 
-{% assign tab_names_html = "Load and replicate;Phased migration;Failback" %}
-{% assign html_page_filenames = "migrate-to-cockroachdb.html;migrate-in-phases.html;migrate-failback.html" %}
-
-{% include filter-tabs.md tab_names=tab_names_html page_filenames=html_page_filenames page_folder="molt" %}
+<div class="filters filters-big clearfix">
+    <button class="filter-button" data-scope="postgres">PostgreSQL</button>
+    <button class="filter-button" data-scope="mysql">MySQL</button>
+    <button class="filter-button" data-scope="oracle">Oracle</button>
+</div>
 
-## Before you begin
+## Prepare the CockroachDB cluster
 
-[Enable rangefeeds]({% link {{ site.current_cloud_version }}/create-and-configure-changefeeds.md %}#enable-rangefeeds) in the CockroachDB SQL shell:
+[Enable rangefeeds]({% link {{ site.current_cloud_version }}/create-and-configure-changefeeds.md %}#enable-rangefeeds) on the CockroachDB cluster:
 
 {% include_cached copy-clipboard.html %}
 ~~~ sql
 SET CLUSTER SETTING kv.rangefeed.enabled = true;
 ~~~
 
-Select the source dialect you migrated to CockroachDB:
+<section class="filter-content" markdown="1" data-scope="oracle">
+## Grant Oracle user permissions
 
-<div class="filters filters-big clearfix">
-    <button class="filter-button" data-scope="postgres">PostgreSQL</button>
-    <button class="filter-button" data-scope="mysql">MySQL</button>
-</div>
+You should have already created a migration user on the source database with the necessary privileges. Refer to [Create migration user on source database]({% link molt/migrate-data-load-replicate-only.md %}?filters=oracle#create-migration-user-on-source-database).
+
+Grant the Oracle user additional `INSERT` and `UPDATE` privileges on the tables to fail back:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+GRANT SELECT, INSERT, UPDATE, FLASHBACK ON migration_schema.employees TO MIGRATION_USER;
+GRANT SELECT, INSERT, UPDATE, FLASHBACK ON migration_schema.payments TO MIGRATION_USER;
+GRANT SELECT, INSERT, UPDATE, FLASHBACK ON migration_schema.orders TO MIGRATION_USER;
+~~~
+</section>
+
+## Configure failback
+
+Configure the MOLT Fetch connection strings and filters for `failback` mode, ensuring that the CockroachDB changefeed is correctly targeting your original source.
+
+### Connection strings
+
+In `failback` mode, the `--source` and `--target` connection strings are reversed from other migration modes: 
+
+`--source` is the CockroachDB connection string. For example:
+
+~~~
+--target 'postgres://crdb_user@localhost:26257/defaultdb?sslmode=verify-full'
+~~~
+
+`--target` is the connection string of the database you migrated from.
+
+<section class="filter-content" markdown="1" data-scope="postgres">
+For example:
+
+~~~
+--source 'postgres://postgres:postgres@localhost:5432/molt?sslmode=verify-full'
+~~~
```

Comment:
I see what this section is doing, but it does seem a bit confusing especially looking at it from the previoew link. Since it talks about source then references target.

Reply:
Actually this is just an oversight. I need to switch these two flags!